src/map_command_handler.hpp

```diff
@@ -191,9 +191,16 @@ class map_command_handler
 		else if (help_on_unknown_) {
 			utils::string_map symbols;
 			symbols["command"] = get_cmd();
-			symbols["help_command"] = cmd_prefix_ + "help";
-			print("help", VGETTEXT("Unknown command '$command', try $help_command "
-				"for a list of available commands.", symbols));
+			std::string input = get_data(0);
+			std::string compare = ":";
+			if (input[0] == compare[0]){
```

Comment:
Is there a reason to declare `compare` rather than just directing checking `:`?

Reply:
I though I had tried it and it gave me an error, but looks like I did something wrong then, it's now cleaner (on 3d4d406).